keyboards/owlab/jelly/config.h

```diff
@@ -0,0 +1,94 @@
+
+#ifndef CONFIG_H
+#define CONFIG_H
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter*/
+#define VENDOR_ID       0x4F53  //0x4F53 OS for owl studio
+#define PRODUCT_ID      0x4A53  //0x4A4C JL for jelly, 0x4A53 JS for jelly solder
+#define DEVICE_VER      0x0001 
+#define MANUFACTURER    OwLab 
+#define PRODUCT         JELLY
+#define DESCRIPTION     A custom 75% Keyboard by OwLab
+
+
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 14
+
+
+#define MATRIX_ROW_PINS { B0, B1, B2, B3, A15, B10 }
+#define MATRIX_COL_PINS { A1, A2, A3, A4, A5, A6, A7, A8, A9, A10, B11, B8, B9, C13 }
+
+
+/* NKRO */
+#ifdef NKRO_ENABLE
+#         define FORCE_NKRO
+#endif
+
+
+#ifdef RGB_MATRIX_ENABLE
+
+#    define USE_I2CV2
+#    define DRIVER_ADDR_1 0b0110000
+#    define DRIVER_ADDR_2 0b0110000
+#    define DRIVER_COUNT 2
+#    define DRIVER_1_LED_TOTAL 30
+#    define DRIVER_LED_TOTAL DRIVER_1_LED_TOTAL //+DRIVER_2_LED_TOTAL)
+
+#    define RGB_MATRIX_DISABLE_KEYCODES   
+#    define RGB_DISABLE_AFTER_TIMEOUT 0 // number of ticks to wait until disabling effects
+
+#    define RGB_DISABLE_WHEN_USB_SUSPENDED true // turn off effects when suspended
+#    define RGB_MATRIX_KEYPRESSES
+#    define RGB_MATRIX_LED_PROCESS_LIMIT 4
+#    define RGB_MATRIX_LED_FLUSH_LIMIT 26
+#    define RGB_MATRIX_MAXIMUM_BRIGHTNESS 200 
+#    define RGB_MATRIX_STARTUP_VAL     128 
+
+
+
+#endif
+
+
+
+
+
+
+#define RGB_DI_PIN B15
+
+// #define WS2812_SPI SPID2
+// #define WS2812_SPI_MOSI_PAL_MODE 5
+
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```

Comment:
Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

Reply:
Hi, all the animations are enabled. Should I define all animations?

Reply:
Hi, all the animations are enabled. Should I define all animations?